src/mscorlib/shared/System/Numerics/Vector.cs

```diff
@@ -763,6 +767,37 @@ private Vector(ref Register existingRegister)
         {
             this.register = existingRegister;
         }
+
+#if netcoreapp
+        /// <summary>
+        /// Constructs a vector from the given span. The span must contain at least Vector'T.Count elements.
```

Comment:
nit: use proper xml formatting (see https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/Span.cs#L90).

`The span must contain at least Vector<typeparamref name="T"/>.Count elements.`

Reply:
Yup. But I kept it same like other instances in the file for consistency. E.g. [here](https://github.com/WinCPP/coreclr/blob/71d5215a2c007f7fe9c944bec47f0ed0fc70fefd/src/mscorlib/shared/System/Numerics/Vector.cs#L374-L377) i.e. below code,

```C#
        /// Constructs a vector from the given array. The size of the given array must be at least Vector'T.Count.
        /// </summary>
        [Intrinsic]
        public unsafe Vector(T[] values) : this(values, 0) { }
```
Do we want to fix this as part of the optimization issue #16745?

Reply:
This can be addressed separately and we can make the changes to the whole file at the same time. Good point about consistency with other places in the file.